api4/api.go

```diff
@@ -100,6 +101,7 @@ func InitApi(full bool) {
 	BaseRoutes.TeamByName = BaseRoutes.Teams.PathPrefix("/name/{team_name:[A-Za-z0-9_-]+}").Subrouter()
 	BaseRoutes.TeamMembers = BaseRoutes.Team.PathPrefix("/members").Subrouter()
 	BaseRoutes.TeamMember = BaseRoutes.TeamMembers.PathPrefix("/{user_id:[A-Za-z0-9]+}").Subrouter()
+	BaseRoutes.TeamStats = BaseRoutes.Team.PathPrefix("/stats").Subrouter()
```

Comment:
For this route can we actually not define it here, but in the `InitTeam()` function in `api4/team.go`?